framework/meta-lib/src/tools/wasm_extractor.rs

```diff
@@ -262,3 +271,137 @@ fn is_mem_grow(code_section: &FunctionBody) -> bool {
 
     false
 }
+
+fn is_whitelisted(op: &Operator) -> bool {
```

Comment:
Even though we are not auto-generating it directly right now, it would be nice to be in a separate file.